src/app/modules/shared-components/components/add-content/add-content.component.html

```diff
@@ -0,0 +1,30 @@
+<alg-sub-section icon="sign-in-alt" label="Add a content">
+  <alg-input [parentForm]="newContentForm" name="title" placeholder="Enter a title to create a new child" [hasClearButton]="true"></alg-input>
```

Comment:
the clear button should be shown only if something has been typed!

Reply:
It was supposed to be already done. Fixed